src/calculateRentalCost.js

```diff
@@ -5,6 +5,23 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERN = 7;
+  const SHORT_TERN = 3;
```

Comment:
There is a typo in the constant name 'SHORT_TERN'. It should be 'SHORT_TERM' to correctly reflect its purpose.